src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/WebProfilerExtension.php

```diff
@@ -54,23 +54,13 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setParameter('web_profiler.debug_toolbar.mode', $config['toolbar'] ? WebDebugToolbarListener::ENABLED : WebDebugToolbarListener::DISABLED);
         }
 
-        $baseDir = array();
-        $rootDir = $container->getParameter('kernel.root_dir');
-        $rootDir = explode(DIRECTORY_SEPARATOR, realpath($rootDir) ?: $rootDir);
-        $bundleDir = explode(DIRECTORY_SEPARATOR, __DIR__);
-        for ($i = 0; isset($rootDir[$i], $bundleDir[$i]); ++$i) {
-            if ($rootDir[$i] !== $bundleDir[$i]) {
-                break;
-            }
-            $baseDir[] = $rootDir[$i];
-        }
-        $baseDir = implode(DIRECTORY_SEPARATOR, $baseDir);
+        $projectDir = $container->getParameter('kernel.project_dir');
```

Comment:
don't get the parameter. Use a parameter reference as argument instead (and btw, you can put it directly in the XML file)

Reply:
good idea, done